tests/director/tests/notifications/fixtures_test.go

```diff
@@ -65,8 +68,18 @@ func assertFormationAssignments(t *testing.T, ctx context.Context, tenantID, for
 		require.Truef(t, ok, "Could not find expectations for assignment with source %q and target %q", assignment.Source, assignment.Target)
 
 		require.Equal(t, assignmentExpectation.State, assignment.State)
-		require.Equal(t, str.PtrStrToStr(assignmentExpectation.Config), str.PtrStrToStr(assignment.Configuration))
-		require.Equal(t, str.PtrStrToStr(assignmentExpectation.Value), str.PtrStrToStr(assignment.Value))
+		expectedAssignmentConfigStr := str.PtrStrToStr(assignmentExpectation.Config)
+		assignmentConfiguration := str.PtrStrToStr(assignment.Configuration)
+		if expectedAssignmentConfigStr != "" && expectedAssignmentConfigStr != "\"\"" && assignmentConfiguration != "" && assignmentConfiguration != "\"\"" {
```

Comment:
because the `require.JSONEq()` fails otherwise. I guess it expects valid JSON. The `""` empty string is valid JSON but most probably the `"\"\""`caused it to fail. That's why we use `JSONEq` only when we have non-empty JSON configuration